panel/src/components/Views/Pages/PageChangesView.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+	<k-panel class="k-panel-inside k-page-changes-view">
+		<header class="k-page-changes-header">
+			<k-button-group>
+				<k-button
+					:link="link"
+					:responsive="true"
+					:text="$t('back')"
+					icon="angle-left"
+					size="sm"
+					variant="filled"
+				/>
```

Comment:
`Suggestion:` When the frontend page itself isn't great at indicating which one it is etc., it's a little hard to know where you're at. I think we could draw from the previous preview mode experiment and display the title of the current page here, maybe even with the dropdown tree navigator from that experiment. Even without it, I think this would help to give more orientation and structure to this view.

Reply:
I totally forgot about that. That's a great idea.